src/main/java/core/WebDriverManager.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package core;
+
+import config.ConfigManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public final class WebDriverManager {
+
+    private static WebDriver driver;
+    private static WebDriverWait driverWait;
+
+    private WebDriverManager() {
+    }
+
+    /**
+     * Returns an instance of the WebDriverManager.
+     *
+     * @return WebDriver
+     */
+    public static WebDriver getDriver() {
+        if (driver == null) {

```

Comment:
This method should be enhanced because it does not look easy to maintain. If we add more browsers the if clauses will increase making it hard to maintain.

Reply:
Done.